parser.y

```diff
@@ -1795,6 +1796,14 @@ CreateIndexStmtUnique:
 	{
 		$$ = true
 	}
+|	"FULLTEXT"
```

Comment:
I don't think add FULLTEXT to CreateIndexStmtUnique is proper. we need to add a new non-terminal symbol like `CreateIndexStmtFullText`

Reply:
Yes. It's better to add new  non-terminal symbols. This can expose what kind of index the statement is creating.

Reply:
Yeah, agree that. Now it's a trick for bypassing syntax error, not full function support.

Add new symbol needs more discussion. Except `FULLTEXT`, there are also other key options like `SPATIAL` or MySQL/TiDB will support other types soon or later.

Reply:
Add FULLTEXT and SPATIAL is ok, but put them under `CreateIndexStmtUnique` is not a good idea.
Maybe we can change `CreateIndexStmtUnique` to `CreateIndexStmtOption`, and the optional value is UNIQUE/FULLTEXT/SPATIAL ?

See also https://github.com/twitter-forks/mysql/blob/master/sql/sql_yacc.yy#L2090

Reply:
Enn, use `CreateIndexStmtOption` is much more better. But TiDB is highly depended on `pingcap/parser`, change existed struct should take an attention.

I'll close this PR, let's find a better way so or later.